config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_storagebuckets.storage.cnrm.cloud.google.com.yaml

```diff
@@ -287,6 +287,23 @@ spec:
                 required:
                 - retentionPeriod
                 type: object
+              softDeletePolicy:
+                description: The bucket's soft delete policy, which defines the period
+                  of time that soft-deleted objects will be retained, and cannot be
+                  permanently deleted. If it is not provided, by default Google Cloud
+                  Storage sets this to default soft delete policy.
+                properties:
+                  effectiveTime:
```

Comment:
@yuwenma this field is an output-only field that gets inserted under `spec`. According to our previous discussion, we don't want to add any output-only spec fields any more - is my understanding correct?

Reply:
This is a workaround to hide output-only spec field but still support it under observed state: #2075 .

Reply:
Thank you @maqiuyujoyce for the quick workaround!